packages/runtime/runtime-definitions/package.json

```diff
@@ -63,7 +63,8 @@
 		"@fluidframework/core-interfaces": "workspace:~",
 		"@fluidframework/driver-definitions": "workspace:~",
 		"@fluidframework/id-compressor": "workspace:~",
-		"@fluidframework/protocol-definitions": "^3.1.0"
+		"@fluidframework/protocol-definitions": "^3.1.0",
+		"@fluidframework/telemetry-utils": "workspace:~"
```

Comment:
Did this pass layer-check??  Doesn't feel right to add this dependency from a `-definitions` package to a `-utils` package.

Reply:
Yes, it passes the check. Since it is not the core defn, these already depends on the utils in layerinfo.json.